terraform/demo/main.tf

```diff
@@ -106,6 +106,25 @@ module "data_share_service" {
 
   admin_alerts_enabled           = false
   database_tunnel_alerts_enabled = false
+
+  admin_login_allowed_ip_blocks = [
```

Comment:
I don't know if this definitely true of the resources, but various of them end up doing a sort at some point, so we might end up with unexpected changes if we don't order this - I'd be tempted to apply it and then see if a plan ends recreating any

Reply:
I think this provider is slightly more intelligent: https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/service/wafv2/ip_set.go#L68

Reply:
Given ☝️ I'm confident enough to merge this and test afterwards, fixing if necessary